packages/connector/src/types/snap.ts

```diff
@@ -25,13 +33,13 @@ export type SnapRequestParams = {
   [RPCMethods.ExportIdentity]: never;
   [RPCMethods.GetIdentity]: never;
 
-  [RPCMethods.SaveCredentials]: SaveCredentialsRequestParams;
+  [RPCMethods.SaveCredentials]: [ChainInfo, ClaimOffer];
```

Comment:
actually it does, if i understand u correctly:

https://github.com/rarimo/rarime/pull/49/files#diff-79a2048c6e8b47e4637b545ee543023fcda8257b820090cd873d055518a53ec9R100

https://github.com/rarimo/rarime/pull/49/files#diff-65fd345d408810772a5f750ca41ad141a08002bba67ccb86f8d3d9ddf83e39afR73

actually, it could be auto passed in base class: https://github.com/rarimo/rarime/pull/49/files#diff-79a2048c6e8b47e4637b545ee543023fcda8257b820090cd873d055518a53ec9R53

but i think we should be more specific in that case.

or, in the other hand - we could `Add` chains `onInstall` hook, and pass `coreChainId` and `targetChainId` parameters

Reply:
Ah, missed the part in the `_getChainInfo()`. All good